components/centraldashboard/public/components/main-page.js

```diff
@@ -90,6 +92,11 @@ export class MainPage extends utilitiesMixin(PolymerElement) {
             hideNamespaces: {type: Boolean, value: false, readOnly: true},
             notFoundInIframe: {type: Boolean, value: false, readOnly: true},
             namespace: {type: String, observer: '_namespaceChanged'},
+            __placeholderImage: {
```

Comment:
It doesn't look like this is being used anymore, right? If so, drop the **/public** segment from the path.

Reply:
Just fixed it, and used this value instead.